extensions/wikia/AdEngine/js/SevenOneMediaHelper.js

```diff
@@ -227,7 +227,7 @@ define('ext.wikia.adEngine.sevenOneMediaHelper', [
 	function generateSoiKeywords() {
 		log('generateSoiKeywords', 'debug', logGroup);
 
-		var i, len, param, val, valIndex, valLen, keywords = [];
+		var i, len, param, val, valIndex, valLen, keywords = [], kruxSegments;
```

Comment:
sorry, here is what I meant:
`var foo, bar,
    falcor = 'hairy';`
initialised vars should be last and on new line
at least this is how I read guidelines